doc/classes/Tree.xml

```diff
@@ -431,7 +431,7 @@
 		<signal name="item_collapsed">
 			<param index="0" name="item" type="TreeItem" />
 			<description>
-				Emitted when an item is collapsed by a click on the folding arrow.
+				Emitted when an item is expanded or collapsed by a click on the folding arrow or through code. (Despite its name, this signal is emitted when an item is expanded too.)
```

Comment:
Bark, bark, bark.
```suggestion
				Emitted when an item is expanded or collapsed by clicking on the folding arrow or through code.
				[b]Note:[/b] Despite its name, this signal is emitted even when an item is expanded.
```
It could be `[b]Important:[/b]`, but it is extremely rare in the class reference to the point where its use is debatable. We'll have to go through a lot of notes when we'll implement our own BBCode tags at some point.

Or, because one may argue the description is quite succinct on its own, the note could be removed. Eh.
```suggestion
				Emitted when an item is expanded or collapsed by clicking on the folding arrow or through code.
```